### PR TITLE
Clarify docs for nodePool.spec.config

### PR DESCRIPTION
**Pull request by enxebre**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: api/v1beta1/nodepool_types.go [hunk 1]
```diff
@@ -96,7 +96,13 @@ type NodePoolSpec struct {
 	// https://github.com/openshift/machine-config-operator/blob/18963e4f8fe66e8c513ca4b131620760a414997f/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L185
 	//
 	// Each ConfigMap must have a single key named "config" whose value is the
-	// JSON or YAML of a serialized MachineConfig.
+	// JSON or YAML of a serialized Resource of for machineconfiguration.openshift.io:
```
Comment (bryan-cox) [inline api/v1beta1/nodepool_types.go:99]: Minor typo here - `of for`.

Comment (enxebre) [reply api/v1beta1/nodepool_types.go]: updated

**Pull request merged**